profile.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/dist/css/styles.css">
+</head>
+<body class="bg-primary">
+    <header class="sticky-top">
+        <nav class="navbar navbar-expand-md bg-light">
+            <div class="container-fluid">
+              <a class="navbar-brand" href="index.html"><img src="images/SoMe.png" class="logo"></a>
+              <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+              </button>
+              <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+                <div class="navbar-nav">
+                  <a class="nav-link" aria-current="page" href="index.html">Home</a>
+                  <a class="nav-link active" href="profile.html">Profile</a>
+                  <a class="nav-link" href="login.html">Log out</a>
+                </div>
+                <form class="form-inline my-2 my-lg-0">
+                    <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
+                </form>
+            </div>
+          </nav>
+    </header>
+    <main>
+        <div class="container">
+            <div class="row pt-5 justify-content-center">
+                <div class="d-flex">
+                    <div class="col-4">
```

Comment:
```suggestion
                    <div class="col-12 col-md-4">
```

Use a combination of column classes to create a responsive layout. At the moment this is not responsive!